roles/openshift_master/tasks/main.yml

```diff
@@ -249,7 +249,7 @@
   # Using curl here since the uri module requires python-httplib2 and
   # wait_for port doesn't provide health information.
   command: >
-    curl --silent
+    curl --silent --tlsv1.2
```

Comment:
Should this be handled in a more dynamic fashion, especially with users being able to specify tls config now?